src/main/target/BEESTORM/config.c

```diff
@@ -0,0 +1,156 @@
+
+#include "flight/mixer.h"
+#include "flight/pid.h"
+
+#include "pg/vcd.h"
+
+#include "rx/rx.h"
+
+#include "io/serial.h"
+#include "io/osd.h"
+
+#include "sensors/battery.h"
+#include "sensors/gyro.h"
+
+#include "telemetry/telemetry.h"
+
+#define BEST_FRSKY_RSSI_CH_IDX     9
+
+#ifdef BRUSHED_MOTORS_PWM_RATE
+#undef BRUSHED_MOTORS_PWM_RATE
+#endif
+
+#define BRUSHED_MOTORS_PWM_RATE 32000           // 32kHz
+
+void targetConfiguration(void)
+{
+    if (hardwareMotorType == MOTOR_BRUSHED) {
+        motorConfigMutable()->dev.motorPwmRate = BRUSHED_MOTORS_PWM_RATE;
+        motorConfigMutable()->minthrottle = 1030;
+        pidConfigMutable()->pid_process_denom = 1;
+    }
+
+    for (uint8_t pidProfileIndex = 0; pidProfileIndex < MAX_PROFILE_COUNT; pidProfileIndex++) {
+        pidProfile_t *pidProfile = pidProfilesMutable(pidProfileIndex);
+
+        pidProfile->pid[PID_ROLL].P  = 86;
+        pidProfile->pid[PID_ROLL].I  = 50;
+        pidProfile->pid[PID_ROLL].D  = 60;
+        pidProfile->pid[PID_PITCH].P = 90;
+        pidProfile->pid[PID_PITCH].I = 55;
+        pidProfile->pid[PID_PITCH].D = 60;
+        pidProfile->pid[PID_YAW].P   = 123;
+        pidProfile->pid[PID_YAW].I   = 75;
+    }
+
+    for (uint8_t rateProfileIndex = 0; rateProfileIndex < CONTROL_RATE_PROFILE_COUNT; rateProfileIndex++) {
+        controlRateConfig_t *controlRateConfig = controlRateProfilesMutable(rateProfileIndex);
+
+        controlRateConfig->rcRates[FD_YAW] = 120;
+        controlRateConfig->rcExpo[FD_ROLL] = 15;
+        controlRateConfig->rcExpo[FD_PITCH] = 15;
+        controlRateConfig->rcExpo[FD_YAW]  = 15;
+        controlRateConfig->rates[FD_ROLL]  = 85;
+        controlRateConfig->rates[FD_PITCH] = 85;
+    }
+
+    batteryConfigMutable()->batteryCapacity = 250;
+    batteryConfigMutable()->vbatmincellvoltage = 28;
+    batteryConfigMutable()->vbatwarningcellvoltage = 33;
+
+    *customMotorMixerMutable(0) = (motorMixer_t){ 1.0f, -0.414178f,  1.0f, -1.0f };    // REAR_R
+    *customMotorMixerMutable(1) = (motorMixer_t){ 1.0f, -0.414178f, -1.0f,  1.0f };    // FRONT_R
+    *customMotorMixerMutable(2) = (motorMixer_t){ 1.0f,  0.414178f,  1.0f,  1.0f };    // REAR_L
+    *customMotorMixerMutable(3) = (motorMixer_t){ 1.0f,  0.414178f, -1.0f, -1.0f };    // FRONT_L
+
+    vcdProfileMutable()->video_system = VIDEO_SYSTEM_NTSC;
+    strcpy(pilotConfigMutable()->name, "BeeStorm");
+    osdConfigMutable()->cap_alarm  = 250;
+    osdConfigMutable()->item_pos[OSD_CRAFT_NAME]        = OSD_POS(9, 11)  | VISIBLE_FLAG;
+    osdConfigMutable()->item_pos[OSD_MAIN_BATT_VOLTAGE] = OSD_POS(23, 10) | VISIBLE_FLAG;
+    osdConfigMutable()->item_pos[OSD_ITEM_TIMER_2]      = OSD_POS(2, 10)  | VISIBLE_FLAG;
+    osdConfigMutable()->item_pos[OSD_FLYMODE]           = OSD_POS(17, 10) | VISIBLE_FLAG;
+    osdConfigMutable()->item_pos[OSD_VTX_CHANNEL]       = OSD_POS(10, 10) | VISIBLE_FLAG;
+    osdConfigMutable()->item_pos[OSD_RSSI_VALUE]         &= ~VISIBLE_FLAG;
```

Comment:
These defaults should not be set - they will just make target maintenance much harder.